cypress/integration/plugins/security-dashboards-plugin/switch_tenant.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export function switchTenantTo(newTenant) {
+  cy.getElementByTestId('account-popover').click();
+  cy.intercept({
+    method: 'GET',
+    url: '/api/v1/auth/dashboardsinfo',
```

Comment:
did we every wanted this configurable? or is the url pretty static (`v1`)

Reply:
but really thank you for intercepting this, this helps make the test less flaky compared to arbitrary wait times